mgr.go

```diff
@@ -338,16 +340,20 @@ func (mgr *SysboxMgr) unregister(id string) error {
 
 	// revert mount prep actions
 	for _, revInfo := range info.mntPrepRev {
-		if revInfo.chown {
-			logrus.Infof("reverting chown on %s to %d:%d for %s", revInfo.path, revInfo.origUid,
-				revInfo.origGid, formatter.ContainerID{id})
+		if revInfo.uidShifted {
+			logrus.Infof("reverting uid-shift on %s for %s", revInfo.path, formatter.ContainerID{id})
+
+			// revInfo.origUid is guaranteed to be higher than revInfo.targetUid
```

Comment:
You probably meant the other way around: revInfo.origUid lower than revInfo.targetUid ...

Reply:
Yes good catch, thanks.